module #3. REST API Advanced/core/src/main/java/com/epam/esm/dao/criteria/CriteriaSet.java

```diff
@@ -0,0 +1,38 @@
+package com.epam.esm.dao.criteria;
+
+import com.epam.esm.dao.criteria.impl.*;
+
+import java.util.Arrays;
+
+public enum CriteriaSet {
```

Comment:
Looks weird. Total number of enum entries is cartesian product of criterias... How can be improved?